src/roles/httpd/templates/foreman-ssl-vhost.conf.j2

```diff
@@ -77,8 +114,12 @@
   ProxyPass /pulp !
   ProxyPass /pub !
   ProxyPass /icons !
+  ProxyPass /images !
   ProxyPass /server-status !
   ProxyPass / {{ httpd_foreman_backend }}/ retry=0 timeout=900
+  ProxyPass /webpack !
+  ProxyPass /assets !
```

Comment:
In our setup we do need to forward these to Puma because we don't serve them via Apache
```suggestion
```